### PR TITLE
update "wordcloud" types to align upstream

### PR DESCRIPTION
**Pull request by g-plane**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test wordcloud`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/timdream/wordcloud2.js/blob/gh-pages/API.md
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

Types of "wordcloud" are out of dated. This PR updated them.

Existing code doesn't match Prettier config, e.g. it originally uses 2-space indentation but Prettier is configured with 4-space indentation. When I save and format the files, file indentation was changed, so you may see unrelated changes.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## wordcloud ([<kbd>unpkg</kbd>](https://unpkg.com/browse/wordcloud@latest/))
was missing the following properties:
1. minFontSize
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 96bd7a0462c4f952a4d2c0fa170f29f32bd47d3f
</p>

File: types/wordcloud/index.d.ts [hunk 1]
```diff
@@ -10,14 +10,17 @@ declare function WordCloud(elements: HTMLElement | HTMLElement[], options: WordC
 
 declare namespace WordCloud {
     const isSupported: boolean;
-    let miniumFontSize: number;
+    const minFontSize: number;
+
+    /** Stop rendering. */
+    function stop(): void;
```
Comment (gabritto) [inline types/wordcloud/index.d.ts:16]: I think this function only exists as of version 1.2, it doesn't seem to be present in version 1.1. If you'd still like to add this function, please update the version these typings correspond to in the header above, like this: `// Type definitions for wordcloud 1.2`

**Review by gabritto — CHANGES_REQUESTED** [review]
Left a comment about updating the version, everything else looks good. Thanks.

**Review by gabritto — APPROVED** [review]

**Comment by g-plane** [comment]
Ready to merge

**Pull request merged**